src/codeu/chat/server/Server.java

```diff
@@ -50,6 +51,8 @@
 
   private static final int RELAY_REFRESH_MS = 5000;  // 5 seconds
 
+  private static final ServerInfo info = new ServerInfo();
```

Comment:
static final variables should be upper case: https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names